src/features/layers/selectors.js

```diff
@@ -69,23 +18,23 @@ export const getKonvaLayerIds = createSelector(
 )
 
 export const isDragging = createSelector(
-  [ getLayerIds, getLayersById ],
-  (layerIds, layers) => {
+  [ getLayerOrder, getLayersById ],
+  (layerOrder, layers) => {
```

Comment:
I don't find getLayerOrder to be intuitive. If you want to convey order, can we go with something like "getOrderedLayerIds" and "orderedLayerIds" throughout?

Reply:
I was getting confused with the old name. This is the list of layer ids, in an order (whereas the byId is just a map and can be in any order). The key piece of information in this list is the order.

getOrderedLayerIds or orderedLayerIds both make a lot of sense to me. I am fine with that.

Reply:
ok, great. getOrderedLayerIds is the method, orderedLayerIds is the variable in the code above.